src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -727,5 +727,103 @@ public override void Initialize(AnalysisContext context)
                 }, SyntaxKind.IdentifierName);
             }
         }
+
+        [Fact]
+        [WorkItem(32661, "https://github.com/dotnet/roslyn/issues/32661")]
+        public void SuppressedTypelessExpressionNullability()
+        {
+            var source = @"
+using System;
+class C
+{
+    void M()
+    {
+        Delegate d1 = () => {}!;
```

Comment:
>Delegate [](start = 8, length = 8)

This is not a delegate type.  I would expect errors on this program.

Reply:
Because otherwise the fact that we get not null could be due to the error.

---
In reply to: [280219873](https://github.com/dotnet/roslyn/pull/35384#discussion_r280219873) [](ancestors = 280219873)